porter/v1/cluster_control_plane.proto

```diff
@@ -417,6 +417,24 @@ message DeploymentTargetDetailsResponse {
   bool is_preview = 3;
 }
 
+// CreateDeploymentTargetRequest is the request object for CreateDeploymentTarget
+message CreateDeploymentTargetRequest {
+  int64 project_id = 1;
+  // name is the name of the deployment target
+  string name = 2;
+  // namespace is the namespace that the deployment target points to
+  string namespace = 3;
```

Comment:
if we're essentially saying that the selector type is guaranteed to be namespace, do we want to just call the selector namespace?